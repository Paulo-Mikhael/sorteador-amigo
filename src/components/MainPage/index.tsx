import Formulario from "../Formulario";

const MainPage = () => {
  return (
    <>
      <Formulario />
    </>
  );
}

export default MainPage;