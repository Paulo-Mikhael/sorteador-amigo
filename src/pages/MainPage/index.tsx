import Formulario from "../../components/Formulario";
import ListaParticipantes from "../../components/ListaParticipantes";
import Rodape from "../../components/Rodape";
import Titulo from "../../components/Titulo";

const MainPage = () => {
  return (
    <>
      <Titulo>
        Vamos Come√ßar!
      </Titulo>
      <Formulario />
      <ListaParticipantes />
      <Rodape />
    </>
  );
}

export default MainPage;