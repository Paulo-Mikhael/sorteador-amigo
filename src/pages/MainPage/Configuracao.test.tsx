import { render } from "@testing-library/react";
import MainPage from ".";
import { RecoilRoot } from "recoil";

const mockNavegacao = jest.fn();

jest.mock('react-router-dom', () => {
  return {
    useNavigate: () => mockNavegacao
  }
});

describe('configuração da Main page', () => {
  test('a pagina principal deve ser assim', () => {
    const { container } = render(<RecoilRoot>
      <MainPage />
    </RecoilRoot>);

    expect(container).toMatchSnapshot();
  });
});