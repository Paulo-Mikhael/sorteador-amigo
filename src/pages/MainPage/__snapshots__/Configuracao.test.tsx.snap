// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configuração da Main page a pagina principal deve ser assim 1`] = `
<div>
  <h1
    class="sc-iGgVNO hZfVeL"
  >
    Vamos Começar!
  </h1>
  <form
    class="sc-aYaIB hNOGfC"
  >
    <label
      for="input-adicionar"
    >
      <img
        alt="adicionar usuário"
        src="images/adicionar.png"
      />
    </label>
    <input
      id="input-adicionar"
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
  </form>
  
  <ul
    class="sc-eqUzNf lcovKd"
  />
  <footer
    class="sc-dcJtft hmBCgn"
  >
    <button
      class="sc-fqkwJk hJxqmN"
      disabled=""
      type="button"
    >
      <img
        alt="ícone de reproduzir"
        src="images/img-botao.png"
      />
      Iniciar brincadeira
    </button>
    <img
      alt="sacolas"
      src="images/sacolas.png"
    />
  </footer>
</div>
`;
